apps/web/next.config.mjs

```diff
@@ -74,6 +74,10 @@ const config = {
         source: '/info/pool/:address',
         destination: '/info/pools/:address',
       },
+      {
+        source: '/mmLinkedPool/:path*',
+        destination: 'https://test.linked-pool.pancakeswap.com/quote-service/:path*',
+      },
```

Comment:
still getting CORS?

Reply:
for now, yes, already ask to BE to add  Access-Control-Allow-Origin on the header, then we can remove this

Reply:
done